### PR TITLE
docs: add @VirginieLemaire as a contributor

### PR DESCRIPTION
**Pull request by VirginieLemaire**
## Description

This PR is part of the Open Sauced Intro to Open Source course

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Doesn't fix an issue

## Mobile & Desktop Screenshots/Recordings

no need (I think)

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by CBID2** [comment]
Thanks for your Pull Request @VirginieLemaire! We've merged in some new PRs since you submitted. Can you merge the latest changes and resolve conflicts? If you need some help with that, check out this post: [Keeping Your Branch Up to Date and Handling Merge Conflicts While Waiting for PR Reviews](https://dev.to/opensauced/keeping-your-branch-up-to-date-and-handling-merge-conflicts-while-waiting-for-pr-reviews-3b3h).

**Comment by VirginieLemaire** [comment]
Thank you @CBID2 this is done

File: .all-contributorsrc
Comment (BekahHW) [inline .all-contributorsrc:910]: You're showing up twice here. We only need one 😄

**Review by BekahHW — CHANGES_REQUESTED** [review]

File: .all-contributorsrc
Comment (VirginieLemaire) [reply .all-contributorsrc:910]: oops, I changed it. Sorry I had not see it 🙈

**Comment by VirginieLemaire** [comment]
Thank you @BekahHW , adjustments made and conflicts resolved with last changes PR's changes

**Comment by VirginieLemaire** [comment]
Hello,
Can somebody help me to understand what I need to do? I applied the changes requested and resolved the conversation but my PR is still waiting for my changes 👀. When I click on the review, I still see the code as it was (with duplication) but I committed changes and now code is clean. I resolve the conflicts each time another PR is merged, and check if code seems ok. I don't know what I need to do more 🤔 
Thank you for your help 🙏🏼

**Comment by BekahHW** [comment]
@VirginieLemaire are you making updates manually or using the CLI tool?

**Comment by VirginieLemaire** [comment]
@BekahHW directly here on GitHub (manually)

**Comment by BekahHW** [comment]
Ah ok. Take a look at the docs, specifically step 6. This updates a couple of things in the readme and ensures the proper layout for how contributors appear on the repo. 

Have you used a terminal before?

**Comment by VirginieLemaire** [comment]
> Have you used a terminal before?
Yes I did

> Take a look at the docs, specifically step 6. This updates a couple of things in the readme and ensures the proper layout for how contributors appear on the repo.
I did work on it before the update, and I think that's why my contrib appaeared twice (1 with the CLI Tool `npm run contributors:add` and another one by committing manually with the terminal).

**Comment by VirginieLemaire** [comment]
Do you want me to go back to a previous commit and start again ?

**Comment by adiati98** [comment]
@VirginieLemaire Let me take a look and review this.
And I'll get back to you :)

File: README.md
Comment (adiati98) [inline README.md:88]: We don't want to remove line 87-88 as these are our newest docs update.

What you need to do:
- Copy and paste these lines and put it back to where the were.

File: README.md
Comment (adiati98) [inline README.md:87]: Remove this line.

File: README.md
Comment (adiati98) [inline README.md:96]: Remove these lines

File: README.md
Comment (adiati98) [inline README.md:214]: Let's remove this line.
It will be regenerated after you run `npm run contributors:generate` command.

File: .all-contributorsrc
Comment (adiati98) [inline .all-contributorsrc:931]: - Move your profile details (line 919-931) to the end of the list, after "Ajiboso-Adeola".
- Delete the comma in line 931.
- Add a comma in the closing bracket's of "Ajiboso-Adeola" on line 945.

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hey @VirginieLemaire,

I left comments for you to fix before we can merge your PR.
Because you need to run `npm run contributors:generate` command, let's make these changes in your terminal :)

To do so:

**On GitHub**

- Go to the `main` branch at your forked repository on GitHub. You can choose `main` branch from the dropdown menu.

  branch forked repo

- Click the "Sync fork", then the "Update fork" button.

  update branch



**In your terminal**

- Go to your working branch, `feature/add-profile-VirginieLemaire`.
- Run `git pull origin main` command.
- Make the changes as suggested in the comments.
- Run `npm run contributors:generate` command.
- Add and commit your changes.
- Push your changes by running `git push`.

Please let me know if you have any questions :)

File: README.md
Comment (VirginieLemaire) [reply README.md:88]: As I synced fork and pull in my local repo, it is like that now in the readme file

File: README.md
Comment (VirginieLemaire) [reply README.md:87]: same than previous comment

File: README.md
Comment (VirginieLemaire) [reply README.md:96]: also updated with sync and pull

File: README.md
Comment (VirginieLemaire) [reply README.md:214]: ok ;)

File: .all-contributorsrc
Comment (VirginieLemaire) [reply .all-contributorsrc:931]: ok

**Comment by VirginieLemaire** [comment]
Thank you @adiati98 ! 
Sorry I stopped for the day because I need to go working 🙈 
I think I understand why I have troubles I didn't have on other repos : I changed my git config to rebase and following all your advises still makes weird things on my local repo. I'm gonna fix my config first and do it again 😅

**Comment by VirginieLemaire** [comment]
ok, it's fixed but the commit history is just a mess. If you prefer me to do all this in a new branch and PR, I'll totally understand 😅

**Review by adiati98 — APPROVED** [review]
@VirginieLemaire thanks for the changes!
Don't worry about the commit history for now. The most important thing is that you learn new things from this process! 🙌

If you haven't, feel free to join our community [Discord](https://discord.com/invite/U2peSNf23P). 😊
Also, check out our [pizza-verse](https://github.com/open-sauced/pizza-verse) repo if you want to contribute further! ✨

you did it GIF

**Pull request merged**
